homeassistant/components/vizio/const.py

```diff
@@ -72,5 +73,7 @@
     ),
 }
 
+VIZIO_ZEROCONF_SERVICE_TYPE = "_viziocast._tcp.local."
```

Comment:
This is only used in tests it seems. Then we can move it to the integration test package.